49-hvac-current-monitor/firmware/platformio.ini

```diff
@@ -0,0 +1,21 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:bw_swan_r5]
+platform = https://github.com/platformio/platform-ststm32.git
```

Comment:
I think you can use `ststm32` without needing the full url

Reply:
Yeah I had to do this because I was getting weird compiler errors (illegal instruction, of all things). Somehow, this fixed it. I'll see if I can go back to `ststm32` without it breaking.